mir-module/src/main/resources/xsl/mods2xMetaDissPlus.xsl

```diff
@@ -786,4 +789,49 @@
     </xsl:choose>
   </xsl:template>
 
+  <xsl:template name="licence">
+    <xsl:param name="derivateID" />
+    <xsl:variable name="mods" select="metadata/def.modsContainer/modsContainer/mods:mods" />
+    <xsl:choose>
+      <xsl:when test="acl:checkPermission($derivateID,'read')">
+        <ddb:licence ddb:licenceType="access">OA</ddb:licence>
+      </xsl:when>
+      <xsl:otherwise>
+        <ddb:licence ddb:licenceType="access">nOA</ddb:licence>
+      </xsl:otherwise>
+    </xsl:choose>
+    <xsl:for-each select="$mods/mods:accessCondition[@type='use and reproduction' and @xlink:href]">
+      <xsl:variable name="licenseId" select="substring-after(@xlink:href,'#')" />
```

Comment:
Actually `licenseId` is correct here, because we're querying the `mir_license`-classification here. The other variable, `licence`, should then be names `license` of course.

The change from "license" (as used by MIR) to "licence" (as used in XMDP) needs to occur somewhere; but at least things should be named consistently.